.travis.yml

```diff
@@ -1,15 +1,21 @@
 cache:
+  yarn: true
   directories:
     - node_modules
-rvm:
-  - 2.2
+language: node_js
+sudo: false
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+node_js:
+  - "5.7"
```

Comment:
Sure, and let's not forget to update the .nvmrc then too. do you want to handle this or can you wait for me to return home?